README.md

```diff
@@ -144,7 +144,7 @@ Both of the extendable classes contain abstract methods that need to be implemen
  
 The Entity class contains a lot of utility methods to make it easier to manage entities, such as removing (and storing) all transitions of FSMs, restoring all transitions of FSMs, and sending animation and state updates.
 Moreover, the `Hkmp.Fsm` namespace contains the `ActionExtensions` class that hosts a lot of methods to execute FSM actions a single time.
-These methods are useful for replicating FSM behaviour by simply getting the action from the FSM and executing it.
+These methods are useful for replicating FSM behavior by simply getting the action from the FSM and executing it.
```

Comment:
This change was not necessary; both spellings were correct, albeit non-American vs American spelling.

Reply:
We can remove this one.